client/make-game.html

```diff
@@ -33,16 +33,23 @@ <h1>Create a new game</h1>
           All players will receive an invitation email with a personal
           link to the game.  The first player will start.
         </p>
+        <fieldset>
+          <legend>Size</legend>
+          <select name="size">
+            <option>Regular</option>
+            <option>SuperScrabble</option>
+          </select>
+        </fieldset>
         <fieldset>
           <legend>Language</legend>
           <select name="language">
             <option>German</option>
             <option>Czech</option>
-            <option>English</option>
+             <option>English</option>
             <option>Estonian</option>
             <option>French</option>
             <option>Hungarian</option>
-            <option>Test</option>
+             <option>Test</option>
```

Comment:
Is "Test" really a language anymore?  Maybe remove that while you're at it.

Reply:
removed.